docs/integrate/index.md

```diff
@@ -0,0 +1,29 @@
+# Integrations
+
+A set of tutorials and guidelines how to integrate CrateDB with 3rd-party
+systems.
+
+
+```{toctree}
+:maxdepth: 1
+
+mongodb
+mysql
+```
+
+
+```{toctree}
+:maxdepth: 1
+
+kafka-connect
+azure-functions
+streamsets
+```
+
+
+:::{tip}
+Please also visit [Overview of CrateDB integration tutorials].
+:::
+
+
+[Overview of CrateDB integration tutorials]: https://community.cratedb.com/t/overview-of-cratedb-integration-tutorials/1015
```

Comment:
Here we can work on the categorization similar to:
<img width="570" alt="Screenshot 2024-02-13 at 17 24 28" src="https://github.com/crate/cratedb-guides/assets/7926726/a5532d4b-3d4e-4726-949c-c1706d77953b">

Reply:
Thanks. See...

- GH-10